src/types/types.ts

```diff
@@ -1218,123 +1218,80 @@ export interface GenericAclCurrentUserScope {
-  cognitefiles_aclAction,
-  cognitefiles_aclScope
->;
-
-export type cogniteusers_aclAction = LIST | CREATE | DELETE;
-export type cogniteusers_aclScope = GenericAclAllScope;
-export type cogniteusers_aclAcl = Acl<
-  cogniteusers_aclAction,
-  cogniteusers_aclScope
->;
-
-export type cogniteprojects_aclAction = LIST | READ | CREATE | UPDATE;
-export type cogniteprojects_aclScope = GenericAclAllScope;
-export type cogniteprojects_aclAcl = Acl<
-  cogniteprojects_aclAction,
-  cogniteprojects_aclScope
->;
+export type RawAclAction = READ | WRITE | LIST;
+export type RawAclScope = GenericAclAllScope;
+export type RawAcl = Acl<RawAclAction, RawAclScope>;
 
-export type cognitesecuritycategories_aclAction =
-  | MEMBEROF
-  | LIST
-  | CREATE
-  | DELETE;
-export type cognitesecuritycategories_aclScope = GenericAclAllScope;
-export type cognitesecuritycategories_aclAcl = Acl<
-  cognitesecuritycategories_aclAction,
-  cognitesecuritycategories_aclScope
->;
-
-export type cogniteraw_aclAction = READ | WRITE | LIST;
-export type cogniteraw_aclScope = GenericAclAllScope;
-export type cogniteraw_aclAcl = Acl<cogniteraw_aclAction, cogniteraw_aclScope>;
-
-export interface CogniteAssetsAclIdScope {
+export interface AssetsAclIdScope {
   assetIdScope: {
     subtreeIds: CogniteInternalId[];
   };
 }
-export type cognitetimeseries_aclAction = READ | WRITE;
-export type cognitetimeseries_aclScope =
-  | GenericAclAllScope
-  | CogniteAssetsAclIdScope;
-export type cognitetimeseries_aclAcl = Acl<
-  cognitetimeseries_aclAction,
-  cognitetimeseries_aclScope
->;
+export type TimeseriesAclAction = READ | WRITE;
+export type TimeseriesAclScope = GenericAclAllScope | AssetsAclIdScope;
+export type TimeseriesAcl = Acl<TimeseriesAclAction, TimeseriesAclScope>;
 
-export type cogniteapikeys_aclAction = LIST | CREATE | DELETE;
-export type cogniteapikeys_aclScope =
-  | GenericAclAllScope
-  | GenericAclCurrentUserScope;
-export type cogniteapikeys_aclAcl = Acl<
-  cogniteapikeys_aclAction,
-  cogniteapikeys_aclScope
->;
+export type ApiKeysAclAction = LIST | CREATE | DELETE;
+export type ApiKeysAclScope = GenericAclAllScope | GenericAclCurrentUserScope;
+export type ApiKeysAcl = Acl<ApiKeysAclAction, ApiKeysAclScope>;
 
-export type cognitethreed_aclAction = READ | CREATE | UPDATE | DELETE;
-export type cognitethreed_aclScope = GenericAclAllScope;
-export type cognitethreed_aclAcl = Acl<
-  cognitethreed_aclAction,
-  cognitethreed_aclScope
->;
+export type ThreeDAclAction = READ | CREATE | UPDATE | DELETE;
+export type ThreeDAclScope = GenericAclAllScope;
+export type ThreeDAcl = Acl<ThreeDAclAction, ThreeDAclScope>;
 
-export type cognitesequences_aclAction = READ | WRITE;
-export type cognitesequences_aclScope = GenericAclAllScope;
-export type cognitesequences_aclAcl = Acl<
-  cognitesequences_aclAction,
-  cognitesequences_aclScope
->;
+export type SequencesAclAction = READ | WRITE;
+export type SequencesAclScope = GenericAclAllScope;
```

Comment:
Also might be better to have `AclScope` and `AclAction` as a prefix. In that case we can put 3d as a postfix and it will be consistent. Also it will be all grouped under `Acl` kinda. But let's discuss